samples/qml/metrics.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.import "../../dist/glean.js" as GleanInstance;
```

Comment:
If we always import glean.js here (and not in the product itself), then we are creating a new requirement: products will be required to at least add one custom metric in order for this to work.

The Glean SDK does not currently have this requirement.

Reply:
This is a great, great point. I let this pass. The elctron PR will have the same issue. I will think about it and if I have a solution will fix it separately. Thanks!